cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -343,6 +349,12 @@ export default class CanvasWrapperComponent extends React.PureComponent<Props> {
         }
         canvasInstance.grid(gridSize, gridSize);
 
+        // Filters
+        const backgroundElement = window.document.getElementById('cvat_canvas_background');
+        if (backgroundElement) {
+            backgroundElement.style.filter = `brightness(${brightnessLevel/50}) contrast(${contrastLevel/50}) saturate(${saturationLevel/50})`;
```

Comment:
Let's setup such border values for:
Brightness: [50 - 200], default 100
Contrast: [50 - 200], default 100
Saturate: [0 - 300], default 100